extensions/azdata/src/services/azdataToolService.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { AzdataTool, IAzdataTool } from '../azdata';
+import * as constants from '../constants';
+
+
+export class AzdataToolService {
+	private _localAzdata: IAzdataTool | undefined;
+	constructor(private _memento: vscode.Memento) {
+		this._localAzdata = this._memento.get<AzdataTool>(constants.localAzdata);
+	}
+
+	/**
+	*	Gets the localAzdata that was last saved
```

Comment:
Changed to spaces. Do we want all our filed to just have spaces for indentations?

Reply:
The indentations should be tabs but the other whitespace should all be spaces generally so it looks consistent

Reply:
I do not see anything weird vscode editor. I even ran a tool to change all indentation to tabs and nothing changed.

Reply:
Changing indentation will do nothing because this isn't indentation - indentation is only the whitespace at the beginning of the line.

You need to delete the tab character between the * and the G and just replace it with a space. I always have "show whitespace" enabled for stuff like this - you can see here what this would look like locally : 

![image](https://user-images.githubusercontent.com/28519865/95131040-4b5b1c00-0712-11eb-92c0-2f227d760b17.png)

(* for a space vs -> for a tab)

Reply:
I do have 'Show Whitespace' enabled. I just missed that you need me to remove the tabs in doc comments and replace it with a ' '.

Reply:
Need is a strong word here 😄 But it does look weird in Github so I'd like that we fix this stuff to at least be consistent with the rest of the comments.

Reply:
Fixed!